core-backend/sandbox/src/funcs.rs

```diff
@@ -800,6 +823,7 @@ where
         let (data_ptr, data_len): (_, u32) = args.iter().read_2()?;
 
         ctx.run(|ctx| {
+            // Todo shall we use Payload here?
```

Comment:
@breathx @ark0f @gshep what do you think?

Reply:
In fact argument for debug should be similarly limited as anything else